src/components/LoadError/index.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const LoadError = ({ error }) => {
+  const { message = error, code = null } = error;
```

Comment:
what does it mean `message = error`?
can error have a string type? if yes, Destructuring assignment will raise an exception.

Reply:
Fixed. Updated in last commit